README.md

```diff
@@ -25,120 +25,163 @@ Today, the attestation model does not seek to provide information to users withi
+   <td><code>IdentifyingUserAcrossSites</code>
+   </td>
+   <td>The act of utilizing the API's data for the purpose of learning the identity of an individual user or device across different sites or apps.
+   </td>
   </tr>
 </table>
 
 The core attestation adheres to several key principles:
 
-* **Flexible coverage:** Ad tech systems are sophisticated and complex, so it's critical that the attestation is flexible enough to cover abuses wherever they might happen: Within the client side API or within any component or connection that makes up the service. 
-* **Practicality:** It's more practical for sites to list commitments for abuse prevention, rather than listing every way a site will use any particular service. As developers become more familiar with these technologies, usage may change.
-* **Focus on outcomes:** Instead of identifying all of the mechanisms by which abuse can occur, the attestation focuses on the result of the abuse (such as, re-identifying users across sites). This simplifies the attestation itself and the description of the attestation, and also allows for comprehension across various audiences, such as regulators, key opinion formers (KOFs), and consumers. 
+**Flexible coverage:** Ad tech systems are sophisticated and complex, so it's critical that the attestation is flexible enough to cover abuses wherever they might happen: within the client side API or within any component or connection that makes up the service. 
+
+**Practicality:** It's more practical for sites to list commitments for abuse prevention, rather than listing every way a site will use any particular service. As developers become more familiar with these technologies, usage may change.
+
+**Focus on outcomes:** Instead of identifying all of the mechanisms by which abuse can occur, the attestation focuses on the result of the abuse, such as re-identifying users across sites. This simplifies the attestation and its description, and allows for comprehension across various audiences such as regulators, key opinion formers, and consumers.
 
 Additional attestations may be added to individual APIs as scenarios arise.
 
 ### Validation and transparency framework
 
-Developers who submit the enrollment form will be sent a file that contains the attestations for the APIs they are requesting to use. To complete enrollment, the developer must place the file in a site's public `.well-known` location. For example, if they enroll with `example.com`, they need to place the attestation file at `https://example.com/.well-known/privacy-sandbox-attestations.json`. 
+Developers who submit an enrollment form are then sent a file that contains the attestations for the APIs they requested to use. To complete enrollment, the developer places the file in a site's public `.well-known` location. For example, if they enroll `example.com`, they need to place the attestation file at `https://foo.com/.well-known/privacy-sandbox-attestations`. 
```

Comment:
Note that as of a week ago, your "enrollment complete" notification to registrants specified the extension. If you are removing it you should notify existing successful registrants who it is reasonable to assume have followed those instructions. 

>28 September 2023
From: Privacy Sandbox Enrollment Support <privacy-sandbox-enrollment@google.com>
>
>Hello,
Thank you for your Privacy Sandbox enrollment. We have completed your enrollment: XXXXXX
If you have not already done so, be sure to upload the attestation file to your site in the .well-known directory ([https://foo.com/.well-known/privacy-sandbox-attestations.json](https://foo.com/.well-known/privacy-sandbox-attestations.json))
>
>Best,
Rob & Privacy Sandbox Enrollment Support